src/components/ui/accordions.js

```diff
@@ -0,0 +1,100 @@
+import { Accordion, Tabs } from 'flowbite-react';
+import { BodyText, LgText } from '@/ui/typography';
+
+/**
+ * @description - this is the basic FAQ accordion adopted from Flowbite-react
+ * @returns
```

Comment:
```suggestion
```
No need to describe the return of this function. However, the function should have its own description.